src/styles.css

```diff
@@ -0,0 +1,1209 @@
+  flex-grow: 0;
+}
+
+.button-click {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-resume {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+  padding-left: 20px;
+}
+
+.btn {
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  padding: 7px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: 5%;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.btn:active {
+  background-color: #2230d2;
+}
+
+.about {
+  background: #fff;
+  width: 70%;
+  padding: 64px;
+  border-top-right-radius: 55px;
+}
+
+.about-me {
+  display: flex;
+  position: static;
+  width: 40%;
+  left: 0.5px;
+  top: 0.5px;
+  right: auto;
+  padding: 15px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40.5px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+}
+
+.lang {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  padding: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.imglang {
+  width: 30px;
+  padding-right: 40px;
+}
+
+.lang-title {
+  position: absolute;
+  height: 24px;
+  left: 12px;
+  right: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20.5px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  padding-left: 24px;
+  justify-content: space-between;
+}
+
+.language-list {
+  padding: 5px;
+}
+
+.lang-skill {
+  position: static;
+  width: 80%;
+  height: 66.72px;
+  left: 12px;
+  top: 12px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  column-gap: 25px;
+  background: #f7f7f9;
+  border-radius: 24px;
+  margin: 9px 0;
+  padding: 12px 12px 12px 12px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.line {
+  display: flex;
+  height: 2px;
+  width: 90%;
+  background: #dfe1e6;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0 0;
+  margin-top: 0%;
+  width: 98%;
+}
+
+.bg-image {
+  background-image: url(../assets/contact-me.png);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-size: contain;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
- [ ] Kindly make sure the background image fits correctly the container in order to match the provided Figma design.

   - ### YOUR DESIGN
   ![image](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/87197412/46f77171-bf0c-4944-9da6-bbcf4549c4c5)

   - ### FIGMA DESIGN
   
    ![image](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/87197412/448390b8-637b-4cf8-887f-563e658eec15)